### DISCUSSION TITLE
culr command line is not opening

### DISCUSSION THREAD

**Discussion by chippy678999** [curl/curl#13114] [General]
I am trying to open curl in windows 11.I see it is installed by default. But it is not opening .Do I need to do any settings for opening curl?

**Comment by jay** [comment]
> Do I need to do any settings for opening curl?

curl is not a Windows GUI application so you wouldn't "open" it. curl is a command line tool that is run with arguments, usually from a script or from a console. If you run it directly, which you can do, Windows will make a separate console and close it when done. If you click on curl.exe then the console Windows creates is closed immediately because no arguments have been provided.

Also (and this is slightly confusing) powershell has its own curl command that is not the same as curl. Instead use the Windows Command Prompt (cmd.exe) and then run curl from that prompt (eg `curl https://example.com`).
